tools/serial_parser.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+"""Automaticly get data from ESP8266's Raw data output and calculating average time for zero, one and space"""
```

Comment:
This is awesome, could we have it in separate PR? And if possible reuse logic between this and the already existing tools?

Reply:
Removed from PR

Reply:
I would still love to see separate PRs for your tools :)

Reply:
> I would still love to see separate PRs for your tools :)

[Initial version of online serial parser without reuse logic from another tools.](https://github.com/crankyoldgit/IRremoteESP8266/pull/2148)